RedHatInsights/vulnerability-ui
# chore(deps): Update FEC and patternfly/react-core

### How to test:
There should be no change in functionality. Just verify there are no visible regressions happening.
